### DISCUSSION TITLE
How to delete LNBank account

### DISCUSSION THREAD

**Discussion by iBobik** [btcpayserver/btcpayserver#4517] [General]
Hi,
There is option to delete wallet, but honestly it is impossible with this requirement:
<img width="615" alt="Snímek obrazovky 2023-01-10 v 11 58 42" src="https://user-images.githubusercontent.com/614232/211533794-8aaab47a-f59e-4d1a-8a59-10b270c6e608.png">

<img width="388" alt="Snímek obrazovky 2023-01-10 v 11 58 59" src="https://user-images.githubusercontent.com/614232/211533859-fa33824d-7b2b-4ba2-990e-f3c8a48b9641.png">

**Comment by Taylor295** [comment]
How do I pay my Bitcoin Bonus Fees

**Comment by dennisreimann** [comment]
Ok, I'll fix that

**Reply by dennisreimann** [reply]
Will be fixed in the next version.

**Comment by iBobik** [comment]
Btw, it would be less pain if I would transfer sats between same node
acounts without reserve for fees, so I can send all.

Then account delete should be possible without transfering money, if I am
ok with keeping them in the node.

10. 1. 2023 13:50:47, d11n ***@***.***> napsal:

**Reply by dennisreimann** [reply]
Internal transfers should be working without a fee reserve. You have to create an invoice from the receiving wallet thought — if that invoice is known internally it should work,

**Comment by iBobik** [comment]
Aha, I used LNURL of the receiving wallet, but it required to keep the
last sat.

10. 1. 2023 14:11:47, d11n ***@***.***> napsal:

**Reply by dennisreimann** [reply]
Does that mean with a regular invoice it worked, just not with LNURL? Maybe I can also make it work for the latter, I have to check.

**Reply by dennisreimann** [reply]
I just checked, for me it works when paying the LNURL of another LNbank wallet. Maybe its because there was > 1000 millisats left?

**Comment by iBobik** [comment]
I have not tried it with regular invoice. Should I?

10. 1. 2023 18:36:18, d11n ***@***.***> napsal:

**Reply by dennisreimann** [reply]
Yeah, please check if this works for you. I'll see if I can make it work with LNURL in one of the next updates.

**Comment by iBobik** [comment]
Yes there are some millisats.Honza Pobořil💬Signal12. 1. 2023 v 16:41, d11n ***@***.***>:﻿
I just checked, for me it works when paying the LNURL of another LNbank wallet. Maybe its because there was > 1000 millisats left?
